README.md

```diff
@@ -99,6 +99,123 @@ Run the following command:
 docker-compose up -d
 ```
 
+#### Docker Swarm w/ Docker Secrets:
+
+This section discusses running an instance of ibeam inside a locked docker swarm, and using the docker swarm facilities for managing secrets.
```

Comment:
Note that this library is actually called IBeam with capital letters 'IB' 😉

Also, Docker Swarm is also a capitalised name.